### PR TITLE
pitivi: pass hicolor-theme to fix missing icons

### PR DESCRIPTION
**Pull request by AkechiShiro**
## Description of changes

Pitivi seems to not be compatible with Adwaita and requires hicolor-theme-icons at runtime.

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [X] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/video/pitivi/default.nix [hunk 1]
```diff
@@ -74,6 +75,13 @@ python3.pkgs.buildPythonApplication rec {
     librosa
   ];
 
+  preFixup = ''
+    gappsWrapperArgs+=(
+      # The icon theme is hardcoded.
+      --prefix XDG_DATA_DIRS : "${hicolor-icon-theme}/share"
+      )
+    '';
```
Comment (wegank) [inline pkgs/applications/video/pitivi/default.nix:83]:
```suggestion
    )
  '';
```

Comment (AkechiShiro) [reply pkgs/applications/video/pitivi/default.nix:83]: Are you using anything to format the nix files ? In order so I can fix this for once and for all ?

Comment (wegank) [reply pkgs/applications/video/pitivi/default.nix:83]: I use nixpkgs-fmt, but the choice shouldn't matter too much. The aim is to ensure consistency of the format in the file, with `postPatch` for example.

**Comment by AkechiShiro** [comment]
This should be ready to get merged @wegank, thanks for your help

**Comment by wegank** [comment]
Yeah, the aarch64-linux failure looks irrelevant.

**Comment by AkechiShiro** [comment]
I noticed I made a little mistake, I added myself to the maintainers but without you know separating the commit, was that okay?

**Comment by wegank** [comment]
It's OK for me.

**Pull request merged**